### DISCUSSION TITLE
Offer a field for a Mastodon profile link in the user profile There's a Twitter profile space in there, but I think GitHub should now add a link for Mastodon accounts, as well.

### DISCUSSION THREAD

**Discussion by alimttth** [community/community#38488] [Other Feature Feedback, Questions, & Ideas] [answered]
as for unlimited links, you can use the default README on your landing page to include all of your links, you can even make it beautiful by making a link icon instead of the actual link  this is what it looks like

**Comment by david-engelmann** [comment]
You can create your own icon and hyperlink to Mastodon or any website. Download a logo image and add your profile as the hyperlink. Checkout how I do it for my personal LinkedIn below.

```
<div align='center'>

Personal LinkedIn       |
:-------------------------:|
[![personal linkedin](https://github.com/david-engelmann/david-engelmann/blob/042ff398c8e5fa7cb5b623b27b02f96a9c4a25f6/assets/LinkedIn_logo_initials.png)](https://www.linkedin.com/in/david-engelmann-217817110)

</div>
```

<div align='center'>

Personal LinkedIn       |
:-------------------------:|
[![personal linkedin](https://github.com/david-engelmann/david-engelmann/blob/042ff398c8e5fa7cb5b623b27b02f96a9c4a25f6/assets/LinkedIn_logo_initials.png)](https://www.linkedin.com/in/david-engelmann-217817110)

</div>

**Comment by kungfuchicken** [comment]
These answers are helpful workarounds, but they are definitely workarounds instead of solutions.

**Reply by jlsjonas** [reply]
Especially as they lack the ability to add `rel="me"` or similar verification methods
(workaround for that is to temporarily override the generic "site" field, though not sure how long the verification would last this way...)

**Comment by derekprior** [comment, marked as answer]
Hey folks, I just wanted to give you a heads up that we are working on this. It'll ship when it's ready, but with holiday season vacations coming up that won't be for a bit yet. Thanks for your patience.
